Library/Formula/slimerjs.rb

```diff
@@ -1,5 +1,6 @@
 class FirefoxDependency < Requirement
   fatal true
+  default_formula "xulrunner"
```

Comment:
Pushed a check. Don't know if the check shells out, which I know Jack has been concerned about in requirements before, but I guess it matters less in a formula? Let me know if we need to get fancier on the check though.